src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/client/processor/BulkScanProcessorApiProxy.java

```diff
@@ -14,7 +13,6 @@
 
 @FeignClient(name = "bulk-scan-processor-api", url = "${bulk-scan-procesor.api.url}",
     configuration = BulkScanProcessorApiProxy.BulkScanConfiguration.class)
-@Profile("!functional")
```

Comment:
why removed?

Reply:
Reinstate it after adding ignore functional profile to configuration files.